### ISSUE TITLE
add simple azmi demo - how to use

### ISSUE THREAD

**Issue by iricigor** [SRE-PRG/azmitool#132]
2 examples: how to read blob and how to read secret

briefly:
- create MI
- create storage account / container, assign Storage Blob Reader , create blob
- or create key vault / secret and assign MI access
- create VM and assign MI
- login to VM and run azmi

**Comment by MangyCoyote** [comment]
listblobs and getblobs should work with Contributor role. One would expect Reader.

**Comment by iricigor** [comment]
Yes, I wanted also to bring security setup in scope of second round of updating testing environment. 
![image](https://user-images.githubusercontent.com/15923171/80949665-b6679600-8df4-11ea-800b-14274879b1c8.png)
